companies/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.1 on 2024-02-13 17:44
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Companies',
```

Comment:
Model names are typically singular in Django, so 'Companies' might be better named as 'Company'.